lib/fastlane/actions/cert.rb

```diff
@@ -0,0 +1,33 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      CERT_FILE_PATH = :CERT_FILE_PATH
+      CERT_CERTIFICATE_ID = :CERT_CERTIFICATE_ID
+    end
+
+    class CertAction
+      def self.run(params)
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `run(*)` if you want the method to accept any arguments but don't care about them.<br>Assignment Branch Condition size for run is too high. [18.49/15]